### PR TITLE
[FIX] remove sequence field in form and tree view in pos_order_note

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

File: pos_order_note/static/src/js/order_note.js [hunk 1]
```diff
@@ -197,27 +208,29 @@ odoo.define('pos_cancel_order.order_note', function (require) {
...
+                            return product.id === current_product;
+                        });
+                    }
+                    if (new_exist_product && curr.custom_notes) {
+                        new_exist_product.custom_notes = curr.custom_notes;
+                    }
+                    if (cancelled_exist_product && curr.old_custom_notes) {
+                        if (curr.old_custom_notes && curr.old_custom_notes !== curr.custom_notes) {
+                            cancelled_exist_product.custom_notes = curr.old_custom_notes;
+                        }
```
Comment (yelizariev) [inline pos_order_note/static/src/js/order_note.js]: Use ``continue`` and don't change indent

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Comment by gustavovalverde** [comment]
@GabbasovDinar, I'll be testing this tomorrow and let you know.

**Comment by yelizariev** [comment]
test

**Review by ilmir-k — APPROVED** [review]
Approved :+1:

**Comment by ilmir-k** [comment]
@yelizariev here are lint warnings related to pos_multi_session_restaurant only, so you can merge it

File: pos_multi_session_restaurant/static/src/js/pos_multi_session_restaurant.js [hunk 2]
```diff
@@ -196,11 +196,15 @@ odoo.define('pos_multi_session_restaurant', function(require){
...
+            var res = OrderlineSuper.prototype.get_line_diff_hash.apply(this, arguments);
+            res = res.split('|');
+            if (res.length > 1) {
+                res[0] = this.uid;
+                res.join('|');
             } else {
-                return '' + this.uid;
+                res = '' + this.uid;
             }
+            return res;
```
Comment (yelizariev) [inline pos_multi_session_restaurant/static/src/js/pos_multi_session_restaurant.js]: you can always use code

                res[0] = this.uid;
                res.join('|');

no need for two cases

File: pos_order_note/static/src/js/tour.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        trigger: ".product-list .product",
+        content: _t("<p>Select the first product in the product list</p>"),
+        position: "bottom"
+    },
+    {
+        trigger: ".control-button:has(.fa-tag)",
+        content: _t("<p>Click on <b>Note</b> button</p>"),
+        position: "bottom"
+    }, {
+        trigger: ".product_note #1",
```
Comment (yelizariev) [inline pos_order_note/static/src/js/tour.js]: having ``id="1"`` in html is a bad idea

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Review by yelizariev — APPROVED** [review]

**Pull request merged**